src/schema/v2/partner_artist.ts

```diff
@@ -58,7 +58,7 @@ const fields: Thunk<
     type: GraphQLBoolean,
     resolve: ({ display_on_partner_profile }) => display_on_partner_profile,
   },
-  isRepresentedBy: {
+  representedBy: {
```

Comment:
the `representedBy ` without the `is` is more heavily used and I wanted one consistent field.

Reply:
We probably adopted that because there are upstream field names that don’t necessarily convey they are predicates. Having said that, the same rules don’t always apply, it should be proper language that is the ultimate goal. To that extend, I think all of the predicate fields have names that could do with some love.

* `isDisplayOnPartnerProfile`: Should probably be `shouldDisplayOnPartnerProfile` or `shouldBeDisplayedOnPartnerProfile` ?
* `isRepresentedBy`: Represented by… whom? As stated, I would expect the value for this key to be some reference to the entity that represents the artist. I assume it’s actually about _this_ partner, so I would have expected the name to be something like `isRepresentedByPartner`.
* `isUseDefaultBiography`: I’m unsure what it means, but at least it should probable have been named something like `shouldUseDefaultBiography`? (To be clear, with that change I am still unclear as to what it means.)

Reply:
> the `representedBy ` without the `is` is more heavily used and I wanted one consistent field.

I don’t follow, what other `representedBy` usage?